scss/_carousel.scss

```diff
@@ -208,12 +218,22 @@
     &:hover,
     &:focus {
```

Comment:
I guess that the weird behavior I can see is due to these selectors : 
- Focus the current page indicator
- There's a semi-moon that rolls over

Reply:
What's the status of this comment? Resolved?